### DISCUSSION TITLE
Funcitions for calculation of principal stress ?

### DISCUSSION THREAD

**Discussion by mora1n** [thelfer/tfel#566] [Q&A] [answered]
Hi @thelfer, 
are there any functions used to calculate principal stress from stress tensor ?
Thx in advance !

**Comment by thelfer** [comment, marked as answer]
Hi @Mora1n,
Most available functions are described here: https://thelfer.github.io/tfel/web/tensors.html
In your case, this section shall help: https://thelfer.github.io/tfel/web/tensors.html#eigenvalues-eigenvectors-and-eigentensors-of-symmetric-tensors

**Reply by mora1n** [reply]
Merci beaucoup !
